src/context/OrderContext.tsx

```diff
@@ -52,9 +61,31 @@ export const LocalOrderProvider: React.FC<Props> = ({ children }) => {
     dispatch({ type: ActionType.RemoveLocalOrder, payload: {} });
   };
 
+  const handleChange = (): void => {
```

Comment:
What is this? Seems unnecessary, bloated, and it shouldn't be in this file. The context is for the dispatch methods.